frontend/src/pages/ClientOverviewPage.tsx

```diff
@@ -77,7 +77,7 @@ const ClientOverview: FC<{
       {
         label: t('Client'),
         keyName: 'clientDotName',
-        value: clientDotName,
+        value: <>{client.name}</>,
```

Comment:
The option for jsx is mostly for styling purposes, so in this case the fragment can be left out
```suggestion
        value: client.name,
```